src/app/app.component.html

```diff
@@ -1,6 +1,6 @@
 <nav>
   <mat-toolbar>
-    <span class="brand">Prylabs</span>
+    <span class="brand">Prysmatic Labs Testnet</span>
```

Comment:
This is going to be too big on mobile. Consider using media queries to serve the larger text for desktop and the smaller one for mobile/tablet

Reply:
Or just leave it as is, it's supposed to be a short little logo, not the title of the site.